draft-ietf-quic-tls.md

```diff
@@ -973,7 +975,7 @@ order are left-padded with zeros to the size of the IV.  The exclusive OR of the
 padded packet number and the IV forms the AEAD nonce.
 
 The associated data, A, for the AEAD is the contents of the QUIC header,
-starting from the flags byte in either the short or long header, up to and
+starting from the initial byte of either the short or long header, up to and
```

Comment:
```suggestion
starting from the first byte of either the short or long header, up to and
```

"initial" is a little loaded.

Reply:
IIRC, I was echoing language from elsewhere, so you might want to see where else you used "initial."